CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.1.0 (2015-06-29)
+
+Features:
+
+  - Add the ability to specify the MAC address of a VM
```

Comment:
Might this say 'set the MAC address'? I think that's more accurate as it implies that it's set during the creation of the VM rather than during some other action - it took me a moment to figure out what this might refer to as-is.